support-frontend/assets/components/subscriptionCheckouts/summary.jsx

```diff
@@ -86,11 +86,15 @@ const DropDownButton = (props: { onClick: Function, showDropDown: boolean }) =>
 
 const TabletAndDesktop = (props: PropTypes) => (
   <span className={styles.tabletAndDesktop}>
-    <div className={styles.img}>
+    <div className={props.product === GuardianWeekly && !props.orderIsAGift ? styles.imgGuardianWeekly : styles.img}>
```

Comment:
Since this: `props.product === GuardianWeekly && !props.orderIsAGift` is repeated it might be better in a variable